tests/unit/test-cmd/test.run.js

```diff
@@ -76,10 +76,9 @@ describe('run', () => {
 
     await cmd.run({startUrl: expectedStartUrls},
                   {FirefoxDesktopExtensionRunner});
-
-    assert.ok(FirefoxDesktopExtensionRunner.called);
-    assert.deepEqual(FirefoxDesktopExtensionRunner.firstCall.args[0].startUrl,
-                     expectedStartUrls);
+    sinon.assert.calledWith(
+     FirefoxDesktopExtensionRunner, sinon.match({startUrl: expectedStartUrls})
```

Comment:
This should be indented two spaces. All code should be indented two spaces but I don't think we have a lint rule for it when it applies to extending function calls.